src/test/java/com/google/research/bleth/simulator/TracingSimulationIT.java

```diff
@@ -21,76 +26,168 @@
 
     private static final IMovementStrategy MOVE_UP = new UpMovementStrategy();
     private static final IMovementStrategy STATIONARY = new StationaryMovementStrategy();
-
-    private static final int BOARD_DIMENSION_EQUALS_TWO = 2;
-    private static final int MAX_ROUNDS_EQUALS_TWO = 2;
-    private static final int NUMBER_OF_BEACONS_EQUALS_ONE = 1;
-    private static final int NUMBER_OF_OBSERVERS_EQUALS_ONE = 1;
     private static final double TRANSMISSION_THRESHOLD_RADIUS_EQUALS_ONE = 1.0;
     private static final int AWAKENESS_CYCLE_EQUALS_TWO = 2;
     private static final int AWAKENESS_DURATION_EQUALS_ONE = 1;
-    private static final AwakenessStrategyFactory.Type FIXES_AWAKENESS_STRATEGY_TYPE =
-            AwakenessStrategyFactory.Type.FIXED;
+    private static final AwakenessStrategyFactory.Type FIXED_AWAKENESS_STRATEGY_TYPE = AwakenessStrategyFactory.Type.FIXED;
```

Comment:
Why do you need this constant at all? Its name adds nothing over just using the value directly.